.github/actions/is-pr-author-member-elastic-org/action.yml

```diff
@@ -14,6 +14,9 @@ outputs:
   result:
     description: 'The result in either true or false'
     value: ${{ steps.is_elastic_pr_author.outputs.result }}
+  author:
```

Comment:
It can be fake, so by pass the check

Reply:
?

Reply:
`result` will either say true if the author is member of `Elastic` or false otherwise.

I needed to export the `author` to help with the following steps in the `undeploy-my-kibana`, as you can see, the `result` is used regardless

Reply:
>GitHub Action to check if the PR author is member of the GitHub organization.

The description is not accurate, it does not check the PR author it checks a User passed as a parameter or by default the user of the PR

Reply:
no

It does check whether the Given PR's author is member of elastic. For such, it uses the PR number as input parameter.

Author is the output

Reply:
ouch!! it is an output